test/JDBC/expected/BABEL-328.out

```diff
@@ -126,6 +126,14 @@ RETURN
 ) 
 GO
 
+CREATE PROC p AS
```

Comment:
1. Let's rewrite the whole BABEL-328 as a form of MVU test including views and procs.
2. Please make sure every parser rules you introduced are tested.

Reply:
Done